eos/references.yaml

```diff
@@ -1446,20 +1651,57 @@ GR:2004A:
   authors: Goldberger, J. and Roweis, S.
   url: http://eprints.pascal-network.org/archive/00001565/
   title: Hierarchical clustering of a mixture model
+GRvDV:2021A:
```

Comment:
Why is this ``GRvDV:2021A`` here? Should all occurences of that that reference be replaced by the correct one, which is ``GRvDV:2022A``?